src/main/java/Example.java

```diff
@@ -0,0 +1,19 @@
+import org.springframework.boot.SpringApplication;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.stereotype.Controller;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+
+@Controller
```

Comment:
I suspect that this annotation can be safely removed, could you try it? I think that `@EnableAutoConfiguration` should be enough.